ARVideoKit/Rendering/RenderAR.swift

```diff
@@ -8,33 +8,37 @@
 
 import Foundation
 import ARKit
-fileprivate var view:Any?
-fileprivate var renderEngine:SCNRenderer!
+
```

Comment:
remove the following from `struct RenderAR` and put it here:
```
private var view: Any?
private var renderEngine: SCNRenderer!
```

Reply:
updated 62f90d0

Reply:
Ah... the code is already committed.

Reply:
@pingchen114, will review it and get back to you soon